components/tournament_ticker/tournament_ticker.lua

```diff
@@ -55,7 +62,15 @@ local STATUS_UPCOMING = {
 local STATUS_ONGOING = {
 	enum = TOURNAMENT_STATUS.ONGOING,
 	isTournamentInStatus = function(tournament)
-		return tournament.sortdate ~= DateExt.defaultDate and tournament.endDate.timestamp < DateExt.getCurrentTimestamp()
+		-- Has eneded
+		if DateExt.getCurrentTimestamp() >= tournament.endDate.timestamp then
```

Comment:
still think we should shift by 1 day here (to include events as ongoing on their last day)